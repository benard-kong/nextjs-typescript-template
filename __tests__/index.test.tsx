import React from "react";
import TestRenderer from "react-test-renderer";
import Index from "../pages/index";

it("renders", () => {
  const tree = TestRenderer.create(<Index />).toJSON();
  expect(tree).toMatchSnapshot();
});
