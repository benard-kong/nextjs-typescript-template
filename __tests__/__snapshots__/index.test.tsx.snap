// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<h1
  className="pages__Title-lrbmwi-0 fFBxIA"
>
  My page
</h1>
`;
